<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ApogeyConfigWidget</class>
 <widget class="QWidget" name="ApogeyConfigWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>470</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string notr="true">ApogeyConfigWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="romDiskGroupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>ROM Disk</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <item>
       <widget class="QCheckBox" name="romDiskEnableCheckBox">
        <property name="text">
         <string>Enable</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="romDiskLabel">
        <property name="maximumSize">
         <size>
          <width>600</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string notr="true">apogey/romdisk512.bin</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="romDiskToolButton">
        <property name="toolTip">
         <string>Select file</string>
        </property>
        <property name="text">
         <string notr="true">...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="sdGroupBox">
     <property name="title">
      <string>SD Adapter (vinxru)</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QCheckBox" name="sdEnableCheckBox">
        <property name="text">
         <string>Enable</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="sdLabel">
        <property name="maximumSize">
         <size>
          <width>600</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string notr="true">apogey/sdcard/</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="sdToolButton">
        <property name="toolTip">
         <string>Select folder</string>
        </property>
        <property name="text">
         <string notr="true">...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="sdosGroupBox">
     <property name="title">
      <string>SD Adapter (SDOS)</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="1">
         <widget class="QRadioButton" name="hwmPvvVv55RadioButton">
          <property name="text">
           <string>HWM_PVV on i8255</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QRadioButton" name="n8vemRadioButton">
          <property name="text">
           <string notr="true">n8vem</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QRadioButton" name="n8vemVv55RadioButton">
          <property name="text">
           <string>n8vem on i8255</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QRadioButton" name="hwmPvvRadioButton">
          <property name="text">
           <string notr="true">HWM_PVV</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QRadioButton" name="msxRadioButton">
          <property name="text">
           <string notr="true">MSX</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="sdosLabel">
        <property name="maximumSize">
         <size>
          <width>600</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string notr="true">rk86/sd_rk86.img</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="sdosToolButton">
        <property name="toolTip">
         <string>Select folder</string>
        </property>
        <property name="text">
         <string notr="true">...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>romDiskEnableCheckBox</tabstop>
  <tabstop>romDiskToolButton</tabstop>
  <tabstop>sdEnableCheckBox</tabstop>
  <tabstop>sdToolButton</tabstop>
  <tabstop>hwmPvvRadioButton</tabstop>
  <tabstop>hwmPvvVv55RadioButton</tabstop>
  <tabstop>n8vemRadioButton</tabstop>
  <tabstop>n8vemVv55RadioButton</tabstop>
  <tabstop>msxRadioButton</tabstop>
  <tabstop>sdosToolButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
